Enable Xe driver

beckwen opened the pull request:
Enable Xe driver for BMG, add kernel configuration

Tracked-On: OAM-131733

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

beckwen committed:
Enable Xe driver

Enable Xe driver for BMG, add kernel configuration and
insmod relevant module

Tracked-On: OAM-131733
Signed-off-by: Kui Wen <kui.wen@intel.com>

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.
